experimental/framework/fluid-static/package.json

```diff
@@ -29,7 +29,7 @@
   "dependencies": {
     "@fluid-experimental/get-container": "^0.42.0",
     "@fluidframework/aqueduct": "^0.42.0",
-    "@fluidframework/common-definitions": "^0.20.1",
+    "@fluidframework/common-definitions": "^0.21.0-0",
```

Comment:
I'd suggest bumping versions independently (i.e. separate PR)

Reply:
Yes... read the PR description 😏  The first change will be to common-definitions, and then it will be months until that's released and the rest can happen.  I just wanted to get a feel for it all.